specification/sbi_atee.adoc

```diff
@@ -1270,15 +1270,15 @@ The possible error codes returned in `sbiret.error` are shown below.
 == TEE Guest Extension (EID 0x54454547)
 The TEE Guest extension supplements the TEE Host extension, and allows TVMs to communicate with TSM.
 A typical use case for this extension is to relay information to the host. TEE-Guest calls cause a
-trap to the TSM which may exit to the host with `scause` set to ECALL, a6 set to FID, `a0`-`a5` set
-to ECALL args.
+trap to the TSM. TSM should do any processing required and then must forward the ECALL to the host with
```

Comment:
Since we are clarifying this, let's add the a7 as the extension id part as well.

Reply:
Fixed. Thanks.